.github/workflows/build.yaml

```diff
@@ -16,6 +16,8 @@
 
 name: Build and Test
 on:
+  # Enable manual trigger
+  workflow_dispatch:
```

Comment:
You might want to get the input for your feature branch. Please see this - https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

Reply:
Where would the inputs be used for this? From my understanding, the manual run would use the workflow version from the feature branch that has the needed changes. Unless I'm missing something

Reply:
So do you just want to run build action manually from main branch? Or do you want to specify which feature branch you want to build ?

Reply:
Yes to the second question. I want to be able to run the workflow from the feature branch. So with the `workflow_dispatch` I can specify which branch to build the workflow off of

Reply:
<img width="571" alt="image" src="https://github.com/radius-project/radius/assets/11863108/134d3d67-e80d-4ca8-8837-77f640d372e9">

It fetches the branches of the upstream repo. Is this expected? don't you want to build the branch in the forked repo?

Reply:
I just approved this PR. but you might need additional input if you want to support branches in forked repo.

Reply:
Ah no, so the feature branch is in the main repo not in a forked repo. So the branches of the upstream repo is correct